### DISCUSSION TITLE
Suggestion QoL: Add A list of Unlocked Cities with their demands

### DISCUSSION THREAD

**Discussion by s-goutrie** [voxeltycoon/issues#414] [UI&UX]
Adding a list of the unlocked cities with their current demands would be very helpful for planning and  finding Cities to deliver to. Right now you have to look for fitting demands all over the map. 

When Clicking on City names it would be nice to open the Cities Overview window or jump there on the map. 

This would make live a lot easier.

Kindest Regards
JaquesGelee

**Comment by candybomberz** [comment]
I would love this aswell.
